docs/source/index.rst

```diff
@@ -96,14 +96,12 @@ Documentation
    :maxdepth: 1
    :caption: User Guides
 
-   reference/local/index
    examples/iterative-dev-project
    reference/interactive-nodes
-   reference/cloud-administration
-   reference/quota
-   reference/logging
+   reference/clouds-setup/index
```

Comment:
In previous comment I was more thinking of adding a section parallel to User Guides:
```
.. toctree::
   :maxdepth: 1
   :caption: Cloud Administration and Usage
```
to hold these 3 pages. This way, an "administrator" will easily see that "oh they have docs for me".

Wdyt?

Reply:
That also makes sense to me. Moved. Thanks!